migrations/021.do.add-retrieval-times.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE retrieval_times (
+  day DATE NOT NULL,
+  miner_id TEXT NOT NULL,
+  task_id TEXT NOT NULL,
+  time_to_first_byte_p50 INT NOT NULL,
```

Comment:
This column name is rather long, how about using the abbreviation TTFB?

```suggestion
  ttfb_p50 INT NOT NULL,
```